spec/controllers/v1.0/workflows_controller_spec.rb

```diff
@@ -501,16 +505,23 @@
     end
 
     it 'returns status code 200' do
-      post "#{api_version}/workflows/resolve", :params => obj_a, :headers => default_headers
+      get "#{api_version}/workflows", :params => obj_a, :headers => default_headers
 
       expect(response).to have_http_status(200)
-      expect(json.first["id"].to_i).to eq(id)
+      expect(json["data"].first["id"].to_i).to eq(id)
     end
 
-    it 'returns status code 204' do
-      post "#{api_version}/workflows/resolve", :params => obj_b, :headers => default_headers
+    it 'returns status code 200' do
+      get "#{api_version}/workflows", :params => obj_b, :headers => default_headers
 
-      expect(response).to have_http_status(204)
+      expect(response).to have_http_status(200)
+    end
+
+    it 'raises an user error' do
+      get "#{api_version}/workflows", :params => obj_c, :headers => default_headers
+
+      expect(response.body).to include("Exceptions::UserError: Invalid resolve params")
```

Comment:
@hsong-rh can we check the errors instead something similar to this
https://github.com/ManageIQ/catalog-api/blob/08cccd4ff0f728bb83c0d3dc02074cafb0da1d1e/spec/requests/approval_request_spec.rb#L25

https://github.com/ManageIQ/catalog-api/blob/08cccd4ff0f728bb83c0d3dc02074cafb0da1d1e/spec/support/requests_spec_helper.rb#L19